src/Integration.UnitTests/Binding/BindingWorkflowTests.cs

```diff
@@ -298,184 +308,63 @@ public void BindingWorkflow_ArgChecks()
         }
 
         [TestMethod]
-        public void BindingWorkflow_InstallPackages_WhenAllProjectsAreCSharp_Succeed()
-        {
-            BindingWorkflow_InstallPackages_Succeed(ProjectSystemHelper.CSharpProjectKind, Language.CSharp);
-        }
-
-        [TestMethod]
-        public void BindingWorkflow_InstallPackages_WhenAllProjectsAreVbNet_Succeed()
-        {
-            BindingWorkflow_InstallPackages_Succeed(ProjectSystemHelper.VbProjectKind, Language.VBNET);
-        }
-
-        private void BindingWorkflow_InstallPackages_Succeed(string projectKind, Language language)
+        public void BindingWorkflow_InstallPackages_Succeeds_SuccessPropertyIsTrue()
```

Comment:
This test and the next test are new. All of the other changes tests below have been moved to the new _NuGetBindingOperationTests.cs_ file.